src/main/java/org/sunbird/digilocker/model/PullURIResponse.java

```diff
@@ -0,0 +1,184 @@
+package org.sunbird.digilocker.model;
+
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
+
+import javax.xml.bind.annotation.XmlElement;
+
+@JacksonXmlRootElement(localName = "PullURIResponse", namespace = "http://tempuri.org/")
+public class PullURIResponse {
+
+    private ResponseStatus responseStatus;
+    private DocDetails docDetails;
+
+    public PullURIResponse() {
+        this.responseStatus = new ResponseStatus();
+        this.docDetails = new DocDetails();
+    }
+
+    @JacksonXmlProperty(localName = "ResponseStatus", namespace = "http://tempuri.org/")
+    public ResponseStatus getResponseStatus() {
+        return responseStatus;
+    }
+
+    public void setResponseStatus(ResponseStatus responseStatus) {
+        this.responseStatus = responseStatus;
+    }
+
+    @JacksonXmlProperty(localName = "DocDetails", namespace = "http://tempuri.org/")
+    public DocDetails getDocDetails() {
+        return docDetails;
+    }
+
+    public void setDocDetails(DocDetails docDetails) {
+        this.docDetails = docDetails;
+    }
+
+    // Inner class for ResponseStatus
+    public static class ResponseStatus {
+        private String status;
+        private String ts;
+        private String txn;
+
+        public ResponseStatus() {}
+
+        public ResponseStatus(String status, String ts, String txn) {
+            this.status = status;
+            this.ts = ts;
+            this.txn = txn;
+        }
+        @JacksonXmlProperty(localName = "Status",isAttribute = true)
+        public String getStatus() {
+            return status;
+        }
+
+        public void setStatus(String status) {
+            this.status = status;
+        }
+
+        @JacksonXmlProperty(isAttribute = true)
+        public String getTs() {
+            return ts;
+        }
+
+        public void setTs(String ts) {
+            this.ts = ts;
+        }
+
+        @JacksonXmlProperty(isAttribute = true)
+        public String getTxn() {
+            return txn;
+        }
+
+        public void setTxn(String txn) {
+            this.txn = txn;
+        }
+    }
+
+    public static class DocDetails {
+        private IssuedTo issuedTo;
+        private String URI;
+
+        public DocDetails() {
+            this.issuedTo = new IssuedTo();
+        }
+
+        @JacksonXmlProperty(localName ="IssuedTo", namespace = "http://tempuri.org/")
+        public IssuedTo getIssuedTo() {
+            return issuedTo;
+        }
+
+        public void setIssuedTo(IssuedTo issuedTo) {
+            this.issuedTo = issuedTo;
+        }
+
+        @JacksonXmlProperty(localName = "URI", namespace = "http://tempuri.org/")
+        public String getURI() {
+            return URI;
+        }
+
+        public void setURI(String URI) {
+            this.URI = URI;
+        }
+    }
+
+    // Inner class for IssuedTo
+    public static class IssuedTo {
```

Comment:
I use inner class because of mapping the xml format i can create different class but their are 6 object which i check for which need to create sperate class